documentation/latest/diagrams/data-synchronization.puml

```diff
@@ -1,4 +1,4 @@
-@startuml
+@startuml data-synchronization
```

Comment:
Now with this change, the v.01 will fail because images are not supported. My fault here.

Maybe we need to think to add https://github.com/yegor256/jekyll-plantuml plugin and do this diagrams inline.

Reply:
Sounds like a good idea to me

Reply:
works for me.